package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-ledger",
-  "version": "5.1.2",
+  "version": "5.1.3",
```

Comment:
Looks like it was already bumped up in another PR, 5.1.2 should be good until we do another release

Reply:
done